alphadia/constants/default.yaml

```diff
@@ -335,8 +335,11 @@ search_output:
   num_samples_quadratic: 50
   # Minimum number of non-missing values required for quantification
   min_nonnan: 3
-  # Enable normalization of label-free quantification values
-  normalize_lfq: True
+
+  # LFQ method for quantification ("directLFQ" or "QuantSelect")
+  normalization_method: directLFQ
```

Comment:
(minor) we usually use small caps, snake case in config. `directlfq` and `quantselect`.